models/WorkspaceUser.py

```diff
@@ -0,0 +1,27 @@
+from google.cloud import ndb
+from models import get_db
+
+
+client = get_db()
+
+
+class WorkspaceUser(ndb.Model):
+    id_workspace = ndb.IntegerProperty()
+    id_user = ndb.IntegerProperty()
+
+    # class methods (ordered by alphabet)
+    @classmethod
+    def create(cls, id_workspace, id_user):
+        with client.context():
+            # check if there's any entries with the same data already
+            id_check = cls.query(cls.id_workspace == id_workspace, cls. id_user == id_user).get()
```

Comment:
@JanVerderber V tej vrstici je ena napaka: med `cls.` in `id_user` ne sme biti presledka.